service/common/src/main/resources/org/apache/polaris/service/banner.txt

```diff
@@ -1,20 +1,22 @@
-                        //\\\/////T\\////\\/////\\\\\\\  //\\                       
-                     //\\\\\\T///////////////////T\\\\\\\T\\\\\                     
-                  //\\\\/////T\//////////\///////T\\\\\T\\\\\\\\                    
-                 //\\\\\/////\\\T////////////////\\\\\\/\\\\\\\\\                   
-,,..,,,..,,,..,//\\\\////////\\\\\\\\\\/////////\\\\\///\\\\\\\\\\,,,..,,..,,,..,,,.
-,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,..,,,.,,,..,,,..,
+                                                                                                              
+ @@                                                                                                           
+   @@                                                                                                         
+    @@@@                                                                                                      
+      @@@@@                     @@             @@@     @@@@@@@@     @@@      @@@@@@@@  @@@    @@@  @@@@@@@@   
+       @@@@@@@@              @@@@             @@@@@    @@@   @@@   @@@@@    @@@@  @@@@ @@@    @@@  @@@        
+         @@@@@@@@          @@@@@             @@@ @@@   @@@   @@@  @@@ @@@  @@@@        @@@@@@@@@@  @@@@@@@    
+          @@@@@@@@@    @@@@@@@              @@@  @@@   @@@@@@@@  @@@  @@@  @@@@        @@@@@@@@@@  @@@@@@@    
+            @@@@@@@@@@@@@@@@@              @@@@@@@@@@  @@@      @@@@@@@@@@  @@@@  @@@@ @@@    @@@  @@@        
+             @@@@@@@@@@@@@@               @@@     @@@@ @@@     @@@     @@@@  @@@@@@@   @@@    @@@  @@@@@@@@   
+               @@@@@@@@@@@                                                                                    
+               @@@@@@@@@@@@               @@@@@@@     @@@@@@@   @@@          @@@     @@@@@@@   @@@   @@@@@@   
+              @@@@@@@@@@@@@@@             @@@@@@@@@  @@@@@@@@@  @@@         @@@@@    @@@@@@@@  @@@  @@@  @@@  
+            @@@@@@@@@@@@@@@@@@@           @@@   @@@ @@@    @@@  @@@        @@@ @@@   @@@  @@@  @@@   @@@@     
+          @@@@@@@@       @@@@@@@@         @@@@@@@@  @@@    @@@  @@@       @@@  @@@   @@@@@@@   @@@     @@@@   
+         @@@@@@              @@@@@@       @@@       @@@@@@@@@   @@@      @@@@@@@@@@  @@@ @@@   @@@ @@@  @@@   
+       @@@@@                   @@@@@@     @@@        @@@@@@@    @@@@@@@ @@@    @@@@  @@@  @@@  @@@  @@@@@@    
+      @@@                         @@@@                                                                        
+                                    @@@@                                                                      
+                                      @@@                                                                     
+                                         @@                                                                   
+                                                                                                               
```

Comment:
nit: remove the trailing new line. Quarkus adds `Powered by Quarkus 3.21.4` (right-justified) automatically. I think it'll look better without the extra empty line.